src/pages/mesh/advanced/secrets.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Secrets management
+description: Learn how to manage secrets for API Mesh for Adobe Developer App Builder.
+keywords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Secrets management
+
+API Mesh for Adobe Developer App Builder allows you to manage secrets or variables for a mesh. You can use these secrets in your mesh configuration file to securely manage sensitive information. When creating or updating a mesh, you can provide a separate file that defines your secrets. API Mesh encrypts secrets using AES 512-bit encryption.
+
+For security purposes, you cannot receive the secrets once you define them when creating or updating a mesh. For example, running an `aio api-mesh get` command returns your mesh with the values defined within the mesh configuration file, such as `{context.secrets.SECRET}` and does not return the actual secret's value.
+
+## Create a `secrets.yaml` file
+
+Create a `secrets.yaml` file that contains your secrets. You can modify the file name as necessary. Secrets management only supports the `yaml` and `yml` file extensions. The file should be in the following format:
+
+```yaml
+secret: "secret-value"
+```
+
+The following example contains a Bash command as a value for the `HOME` secret. API Mesh supports Bash commands with or without brackets, such as `$HOME` or `$[HOME]`.
+
+<InlineAlert variant="info" slots="text"/>
+
+Bash commands are not supported in Windows environments.
+
+```yaml
+AEM: 
+  TOKEN: "my-token"
+  AEMUSERNAME: "user-name"
+  adminname: "admin-name"
+
+HOME: '\${HOME}'
+```
+
+<InlineAlert variant="warning" slots="text"/>
+
+You cannot escape the `$` character in secrets files. This means that a secret's value cannot contain `$`, unless it is part of a Bash command.
+
+## Add secrets to your mesh configuration file
```

Comment:
Worth mentioning where it can be used:

1. operationa headers
2. local hooks
3. additional resolvers

In case of operationheaders it should be used using the template literals syntax `{context.secrets.<SECRET_NAME>}`

In case of js files (hooks, resolvers), it can be taken from context like `const secretValue = context.secrets.<SECRET_NAME>`